internal/common/crypto_policy.go

```diff
@@ -0,0 +1,126 @@
+package common
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	ocpv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/crypto"
+	"kubevirt.io/ssp-operator/api/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type SSPTLSOptions struct {
+	MinTLSVersion      string
+	OpenSSLCipherNames []string
+}
+
+func (s *SSPTLSOptions) IsEmpty() bool {
+	return len(s.OpenSSLCipherNames) == 0 && s.MinTLSVersion == ""
+}
+
+func (s *SSPTLSOptions) MinTLSVersionId() (minTlsVersionId uint16) {
```

Comment:
The named return is not used and could be simplified to just the type that is returned.

Reply:
right!